qiskit/quantum_info/states/stabilizerstate.py

```diff
@@ -557,7 +560,7 @@ def _get_probablities(self, qubits, outcome, outcome_prob, probs):
         for i in range(len(qubits)):
             qubit = qubits[len(qubits) - i - 1]
             if outcome[i] == "X":
-                is_deterministic = not any(ret.clifford.stabilizer.X[:, qubit])
+                is_deterministic = not any(ret.clifford.stabilizer_x_tableau()[:, qubit])
```

Comment:
We should calculate `ret.clifford.stabilizer_x_tableau()` once, outside the loop, and use that.